<?xml version="1.0"?>
<launch>
    <!--
    Setting this parameter to create a toolbar of name "handeye with a double button, take and remove,
    and another one with name foo, a single button spam and a double button, bar and baz
    -->

    <!-- <rosparam ns="iiwa" param="toolbarSpecifications" >
        ['spec','handeye','take,remove', 'spec', 'foo', 'spam', 'bar,baz']
    </rosparam> -->
    
    <!-- position unit 'm' rotation unit 'rad/s' -->
    <param name = "direction_x_add" type = "double" value = "0" />
    <param name = "direction_y_add" type = "double" value = "-0.2" />
    <param name = "direction_z_add" type = "double" value = "0" />
    <!-- juedui or xiangdui -->
    <param name = "rotation_x" type = "double" value = "0" />
    <param name = "rotation_y" type = "double" value = "0" />
    <param name = "rotation_z" type = "double" value = "0" />
    <!-- CartesianPose (=1) or CartesianPoseLinear (=0) -->
    <param name = "command_cartisian_mode" type = "double" value = "1" />

    <node pkg="iiwa_ros" type="iiwa_ros_cartesian_gauncong" name="CartesianPoseState_1569786053275419313" output="screen" />

</launch>
